src/app/components/Search/search-utils.ts

```diff
@@ -121,27 +128,21 @@ export function isSearchValid(searchTerm: string) {
 export const getSearchTermFromRequest = (request: Request) =>
   new URL(request.url).searchParams.get('q')?.trim() ?? ''
 
-export const searchParamLoader = async ({ request, params }: LoaderFunctionArgs) => {
-  const { network } = params
-  if (!!network && !RouteUtils.getEnabledNetworks().includes(network as Network)) {
+export const useParamSearch = () => {
+  const network = useNetworkParam()
+
+  if (!!network && !RouteUtils.getEnabledNetworks().includes(network)) {
     throw new AppError(AppErrors.InvalidUrl)
   }
 
-  const searchTerm = getSearchTermFromRequest(request)
+  const searchTerm = useSearchParams()[0].get('q')?.trim() ?? ''
```

Comment:
Not sure I get this change, as `getSearchTermFromRequest` is still available.

Reply:
But request isn't available in a hook

Reply:
Ah, this is a different function, overlooked rename `seachParamLoader` -> `useParamSearch`